src/main/java/com/solidbrain/russads/configuration/ConfigurationHelper.java

```diff
@@ -0,0 +1,35 @@
+package com.solidbrain.russads.configuration;
+
+import java.util.List;
+
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+
+import org.springframework.transaction.annotation.Transactional;
+
+import com.solidbrain.russads.configuration.model.ConfigurationProperty;
+
+public class ConfigurationHelper {
+
+	public static final String AUTO_SALES_REP_KEY = "autoSalesRep";
+
+	@PersistenceContext
+	EntityManager entityManager;

```

Comment:
When using Spring Data it is very unusual to inject `EntityManager`. Why you do not just inject `ConfigurationPropertyRepositiry` here?